### PR TITLE
Add project shortname to home page project tiles

### PR DESCRIPTION
**Pull request by Kashif-S**
Jira: GLT-2362

### CODE REVIEW

**Comment by sqwci** [comment]
SonarQube analysis reported 2 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![CRITICAL][CRITICAL] [DashboardHelperService.java#L72](https://github.com/TGAC/miso-lims/blob/bcb1be631c5d1e4211ed50cd7417eb4099afc5ab/spring/src/main/java/uk/ac/bbsrc/tgac/miso/spring/ajax/DashboardHelperService.java#L72): Define a constant instead of duplicating this literal "</b><br/>" 5 times. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)
1. ![CRITICAL][CRITICAL] [DashboardHelperService.java#L74](https://github.com/TGAC/miso-lims/blob/bcb1be631c5d1e4211ed50cd7417eb4099afc5ab/spring/src/main/java/uk/ac/bbsrc/tgac/miso/spring/ajax/DashboardHelperService.java#L74): Define a constant instead of duplicating this literal "Alias" 4 times. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)

[CRITICAL]: https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL'

File: spring/src/main/java/uk/ac/bbsrc/tgac/miso/spring/ajax/DashboardHelperService.java [hunk 1]
```diff
@@ -81,6 +81,23 @@ private StringBuilder generateDashboardCell(StringBuilder b, String misoClass, L
     return b;
   }
 
+  private StringBuilder generateProjectDashboardCell(StringBuilder b, String misoClass, Long id, String name, String alias,
+      String shortName, String... aliasAlternative) {
+    b.append("<a class=\"dashboardresult\" href=\"/miso/" + misoClass + "/" + id
+        + "\"><div onMouseOver=\"this.className=&#39dashboardhighlight&#39\" onMouseOut=\"this.className=&#39dashboard&#39\" class=\"dashboard\">");
+    b.append("Name: <b>" + name + "</b><br/>");
+    b.append("Short Name: <b>" + shortName + "</b><br/>");
+    if (aliasAlternative == null || aliasAlternative.length == 0) {
```
Comment (callunity) [inline spring/src/main/java/uk/ac/bbsrc/tgac/miso/spring/ajax/DashboardHelperService.java]: What sorts of alias alternatives might be expected?
The Short Name line should probably only be displayed if `shortName` is not null (it's an OICR-ism that we require but is optional for other MISO instances)

Comment (Kashif-S) [reply spring/src/main/java/uk/ac/bbsrc/tgac/miso/spring/ajax/DashboardHelperService.java]: I grabbed the alias alternative code from the old `generateDashboardCell` function, although I suppose it won't be necessary if this function is going to be specifically acting on pools.

**Comment by sqwci** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![CRITICAL][CRITICAL] [DashboardHelperService.java#L72](https://github.com/TGAC/miso-lims/blob/a8d7c004068f298d5f60adc96173555bf3c5044f/spring/src/main/java/uk/ac/bbsrc/tgac/miso/spring/ajax/DashboardHelperService.java#L72): Define a constant instead of duplicating this literal "</b><br/>" 5 times. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)

[CRITICAL]: https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL'

**Pull request merged**